packages/next/package.json

```diff
@@ -270,7 +270,7 @@
     "uuid": "8.3.2",
     "vm-browserify": "1.1.2",
     "watchpack": "2.4.0",
-    "web-vitals": "3.0.0-beta.2",
+    "web-vitals": "3.0.0",
```

Comment:
Any reason why this package was updated? I would expect that to be a separate PR.

Reply:
There's a undefined type in 3.0.0-beta.2 which wasn't caught by typescript 4.7.